specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -4051,12 +4050,33 @@
         }
       }
     },
+    "PaginatedLinkedWorkspaceList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning Linked Workspace objects wrapped in ARM resource envelope.",
+      "required": [
+        "nextLink"
+      ],
```

Comment:
Ah... my apologies. I just check [the documentation](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable), there is no requirement that nextlink should be required. Could you please remove this?